src/production.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const history = require('connect-history-api-fallback')
+const { resolve } = require('path');
+const cors = require('cors')
+
+const app = express();
+app.use(cors())
+
+// Setup the frontend if in production (source: https://dennisreimann.de/articles/vue-cli-serve-express.html)
+// Since the frontend is only a set of static pages once built we can serve them as static files on a server
+const publicPath = resolve(__dirname, '../dist')
+const staticConf = { maxAge: '1y', etag: false }
+app.use(express.static(publicPath, staticConf))
+app.use('/', history())
+var config = require('./config.js');
```

Comment:
Forgot to add the TravisCI env check like the backend has, if there is a CI issue bc of this, we should add the env check here

Reply:
There won't be any issues since TravisCI doesn't run `npm run production`